test/services/link/index.test.js

```diff
@@ -15,7 +23,122 @@ describe('link service', () => {
 		this.timeout(10000);
 
 		const url = 'https://schul-cloud.org/';
-		return service.create({ target: url })
+		return service.create({ target: url, forceNew })
```

Comment:
forcenew is only used here, to set it true here :)

Reply:
Is that an issue? I found it to be nicer than `forceNew: true` in every `service.create`